include/core/thermal_expansion_model.h

```diff
@@ -18,6 +18,7 @@
 #define lethe_thermal_expansion_model_h
 
 #include <core/physical_property_model.h>
+#include <core/specific_heat_model.h>
```

Comment:
I would move the phase change stuff out of the specific heat model, there is no reason why thermal expansion should depend on specific heat.

Reply:
I use the function calculate_liquid_fraction which is defined in the specific heat model. Should I move this function to utilities?

Reply:
If you can yeah. You could even define a phase_change.h file in core that is used to house these functions, that would be even cleaner.